### DISCUSSION TITLE
PC Client

### DISCUSSION THREAD

**Discussion by afridymhamudework** [zepp-health/zeppos-forum#354] [Developer Feedback]
Wasted that amount of cash yet doesn't have a pc Client. can't even install any emulator to use it

**Comment by MyoungXUE** [comment]
We have a PC emulator, may I ask what problem you encountered?

https://docs.zepp.com/docs/guides/quick-start/simulator-dev/

**Reply by Savelij-creator** [reply]
By the way, why do I get an error in the emulator that there is no norflash.bin file
